addon-test-support/index.js

```diff
@@ -89,13 +89,14 @@ export function setupTestContainer() {
   }
 
   let params = QUnit.urlParams;
-  let containerVisibility = params.nocontainer ? 'hidden' : 'visible';
 
   if (params.devmode || params.fullscreencontainer) {
-    testContainer.className = ' full-screen';
+    testContainer.classList.add('ember-testing-container-full-screen');
   }
 
-  testContainer.style.visibility = containerVisibility;
+  if (params.nocontainer) {
+    testContainer.classList.add('ember-testing-container-hidden');
```

Comment:
First simply used a `hidden` class, but that is such a commonly used utility class that I switched to using `ember-testing-container-hidden` to avoid conflicts. I renamed the `full-screen` class to `ember-testing-container-full-screen` for the same reason and consistency.